manifests/install.pp

```diff
@@ -284,8 +284,11 @@
         $pip_package  = 'pip'
         $pip_provider = 'pip'
       } else {
+        $pip_package = fact('os.release.major') ? {
```

Comment:
Can we use here the $facts-Hash? I know everybody has his favor way to access the facts. But just to be consistent within the module I would like to see the $facts-Hash.

Reply:
Used fact hash, `$fact['os']['release']['major']`.